wgpu-types/src/lib.rs

```diff
@@ -2992,6 +3000,30 @@ pub struct ImageCopyTexture<T> {
     pub origin: Origin3d,
 }
 
+/// Subresource range within an image, used for clear_image
+#[repr(C)]
+#[derive(Clone, Debug)]
+#[cfg_attr(feature = "trace", derive(serde::Serialize))]
+#[cfg_attr(feature = "replay", derive(serde::Deserialize))]
+pub struct ImageSubresourceRange {
+    /// Included aspects: color/depth/stencil
+    pub aspects: TextureAspect,
+    /// First mipmap level in this subresource
+    pub level_start: u8,
```

Comment:
Let's follow the naming/comments of `struct TextureViewDescriptor` here. We should go even further and use this structure as a member of `TextureViewDescriptor`.